Formula/haskell-language-server.rb

```diff
@@ -0,0 +1,44 @@
+class HaskellLanguageServer < Formula
+  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  homepage "https://github.com/haskell/haskell-language-server"
+
+  url "https://github.com/haskell/haskell-language-server.git",
+    tag:      "1.0.0",
+    revision: "4cd1cf934638881e52b3eba9f70157a4b799c0e9"
+
+  license "Apache-2.0"
+
+  head "https://github.com/haskell/haskell-language-server.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  pour_bottle? do
+    reason "Need help with that ^^'"
+    satisfy { false }
+  end
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
+
+  def install
+    system "cabal", "v2-update"
+    # taken from haskell-language-server/install/src/Cabal.hs
+    system "cabal", "v2-install",
+      "exe:haskell-language-server",
+      "exe:haskell-language-server-wrapper",
+      "-w", "ghc",
+      "--write-ghc-environment-files=never",
+      "--max-backjumps=5000",
+      "--overwrite-policy=always",
+      "--project-file=cabal.project",
```

Comment:
Have you tried leaving these build flags out? I don't think they're necessary. Some of these (e.g. `--max-backjumps`) is already defined in `std_cabal_v2_args`.

Reply:
Will try.

Although this is taken directly from `haskell-language-server`'s build script. I assume that this is the way to "guarantee" correct compilation results; not sure if modifying it will not break the binary in some way.

Reply:
Calling `cabal` in the source directory should already invoke their build scripts, so it really should not be necessary.

Reply:
Seems it works 🎉 
I was convinced that I had already tried that without success. Good you proposed this change 🙇